test/plugins/optional-auth/src/test/resources/rest-api-spec/test/optional-auth/10_basic.yaml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
if we do keep on using the headers, I think we should add a skip: headers section cause not all runners support them as far as I know.